_layouts/main.html

```diff
@@ -1,18 +1,19 @@
 ---
 layout: default
 ---
-<p>{{ page.description_1 | markdownify }}</p>
-<p>{{ page.description_2 | markdownify }}</p>
+
+<p>{% t page.description_1 %}</p>
+<p>{% t page.description_2 %}</p>
 
 <div class='projects'>
-  <h2>{{ page.projects_title }}</h2>
+  <h2>{% t page.projects_title %}</h2>
 
   <div class='project'>
     <div class='logo'>
       <img src='{{ "/assets/img/projects/katuma.png" | prepend:site.baseurl }}' width='140' height='140' />
     </div>
     <div class='description'>
-      {{ page.katuma_description | markdownify }}
+      {% t page.katuma_description %}
```

Comment:
Sadly, `markdownify` doesn't work with `{% %}` tags so we need to write these translations in HTML instead.

Reply:
What if you markdownify in an `assign` variable and then use it? Would that work?

Reply:
I tried and it doesn't. Looks like you can't user filters there either.

Reply:
So do a helper in Ruby that use `markdownify`. I think Jekyll has helpers a la Rails

Reply:
Yes, I thought about that. Or even open a PR against that plugin to allow for this but I don't think it's a big deal now. We were just markdownifying couple links and a <strong>.

We can create an issue for it in the future though.